src/proc/Vigad.ts

```diff
@@ -153,6 +156,33 @@ export class Vigad {
                                 }
                             }
                         );
+                        // Uploading Data to the server
+                        const { isSessionActive } = useSession();
+                        const { get, post } = useAPI();
+                        const {
+                            streamData,
+                            streamRegexAndCaptureAreaSettings,
+                        } = useUploadData();
+
+                        // Check whether the user streamData or streamRegexAndCaptureAreaSettings is enabled and if the session is active at all
+                        if (
+                            isSessionActive.value &&
+                            (streamData.value ||
+                                streamRegexAndCaptureAreaSettings.value)
+                        ) {
+                            for (let i = 0; i < result.length; i++) {
+                                await post(
+                                    `session/abc/data/ca/${result[i].ca_id}`,
```

Comment:
lol, are we all blind? "abc" of course needs to be replaced with the session token